hystrix-turbine-sample/Readme.md

```diff
@@ -0,0 +1,74 @@
+# Use Circuit Breaker Dashboard with Azure Spring Cloud
+[Spring Cloud Netflix Turbine](https://github.com/Netflix/Turbine) is widely used to aggregates multiple [Hystrix](https://github.com/Netflix/Hystrix) metrics streams so that it can be monitored in a single view using Hystrix dashboard. In this tutorial, we will demonstrate how to use them on Azure Spring Cloud.
+
+>[!NOTE] 
+> Netflix Hystrix is widely used in many existing Spring Cloud apps but it is no longer in active development. If you are developing new project, you can consider using [resilience4j](https://github.com/resilience4j/resilience4j) instead. Circuit Breaker Dashboard for resilience4j dependents on micrometer, which is currently not supported for Azure Spring Cloud, thus will not be covered by this tutorial.
```

Comment:
"you can consider using resilence4j" -> "you should use Spring Cloud Circuit Breaker".

Reply:
hystrix is an implementation of Spring Cloud Circuit Breaker as well so i think it's not accurate to say that. see: https://spring.io/projects/spring-cloud-circuitbreaker

Does "you should use developing Spring Cloud Circuit Breaker implementations like resilence4j instead" sounds better?

Reply:
I am little confused by Spring Cloud Circuit Breaker project now.
My previous understanding is that the new Spring Cloud Circuit Breaker will support multiple detail implementations. Hystrix and Resilience4J are two of them. Customers should be able to migrate to the new Spring Cloud Circuit Breaker apis.
But from the latest docs, it looks like different from my previous understanding.

I think the key problem is metrics data pipeline. The legacy Hystrix library has its own pipeline. Now the new Spring Cloud Circuit Breaker framework unifies all implementations into Micrometer. We should emphasize this part to customers. 
But I didn't see any Spring Cloud docs mentioning about the backward compatibility to Hystrix.

Reply:
that's a good comment on micrometer,  changed phrases in this note.

Reply:
"If you are developing new project, you should use _developing_ Spring Cloud Circuit Breaker implementation" -> The second "developing" is not needed?
Rest of the paragraph looks good.

Reply:
done